docs/Project-Board-Guide.md

```diff
@@ -0,0 +1,22 @@
+# Simorgh Project Board Guide
+
+Overview of a Github Project: https://help.github.com/en/articles/about-project-boards
+
+BBC's Simorgh Project board: https://github.com/orgs/bbc/projects/20
+
+### Overiew of each column
+
+<!-- prettier-ignore -->
+| Column Name               | Issues/Pull Requests that belong in this column | When to move card to the next column | Additional context |
+| ------------------------- | ------------------------------------------------| ------------------------------------ |------------------- |
+| To-do | Contains issues that have been newly created and added to the Simorgh project. | Once the issue has been refined to the corresponding team that has been labelled to the issue, then it can be manually moved to the "Ready for Dev". | n/a |
```

Comment:
```suggestion
| To-do | Contains issues that have been newly created and added to the Simorgh project. | Once the issue has been refined to the corresponding team that has been labelled to the issue, then it can be manually moved to "Ready for Dev". | n/a |
```